test/e2e/suite/command/manifest.go

```diff
@@ -55,6 +56,8 @@ var _ = Describe("ORAS beginners:", func() {
 			It("should show help doc with feature flags", func() {
 				out := ORAS("manifest", "push", "--help").MatchKeyWords(ExampleDesc).Exec()
 				gomega.Expect(out).Should(gbytes.Say("--distribution-spec string\\s+%s", regexp.QuoteMeta(feature.Preview.Mark)))
+				// verbose flag should be hidden in help doc
+				gomega.Expect(out).ShouldNot(gbytes.Say("--verbose"))
```

Comment:
1. out.Out should be used for matching
2. Suggest adding new spec for verbose flag checking. The reason is out.Out is not reset after matching preview feature mark. You will get false positive when `--verbose` is printed before `--distribution-spec` flag.

Reply:
Updated.